src/main/java/org/helpberkeley/memberdata/RestaurantBean.java

```diff
@@ -40,6 +44,7 @@ public interface RestaurantBean {
     String getControlBlockKey();
     String getControlBlockValue();
     String getRoute();
+    String[] getFormulas();
```

Comment:
Change to List<String>.  Unless you have strong need to use array (hardware mapping, protocol mapping, lowest level performance) prefer using List, Set, Map, etc... They are more more flexible and easier to use data structures than arrays.